### PR TITLE
chore: add two comparator

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

File: ci/azure-pipelines.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
       - checkout: self
       - script: make unit-test
         displayName: Run unit test
-      - script: bash <(curl https://codecov.io/bash) -t $CODECOV_UPLOAD_TOKEN
```
Comment (troyronda) [inline ci/azure-pipelines.yml:36]: ?

Comment (fqutishat) [reply ci/azure-pipelines.yml:36]: no need for CODECOV_UPLOAD_TOKEN

Comment (fqutishat) [reply ci/azure-pipelines.yml:36]: code cov will work with azure

Comment (fqutishat) [reply ci/azure-pipelines.yml:36]: Note: Token not required for repositories uploading from Travis, CircleCI, AppVeyor, Azure Pipelines or GitHub Actions.

**Pull request merged**